Content.Shared/Inventory/InventorySystem.Relay.cs

```diff
@@ -18,7 +18,10 @@ public void InitializeRelay()
 
     protected void RelayInventoryEvent<T>(EntityUid uid, InventoryComponent component, T args) where T : EntityEventArgs
     {
-        var containerEnumerator = new ContainerSlotEnumerator(uid, component.TemplateId, _prototypeManager, this);
+        // Insulated gloves in you pockets ain't gonna save you.
+        var flags = ~SlotFlags.POCKET; 
+
+        var containerEnumerator = new ContainerSlotEnumerator(uid, component.TemplateId, _prototypeManager, this, flags);
```

Comment:
Would this not make more sense as some sort of opt-out? I can imagine scenarios where you'd want to relay events to in-pocket items.

Reply:
Well you could opt out by just not using the generic `RelayInventoryEvent<T>()` subscription. But if there should be a generic way of having each event specify its own target slots, then I think you'd have to do it by adding some sort of `IInventoryRelayEvent` interface that those events implement which requires them to specify target slots? Unless someone can think of something better I can add that.

Reply:
I've now just added an `IInventoryRelayEvent`